src/modality-specific-files/microelectrode-electrophysiology.md

```diff
@@ -0,0 +1,672 @@
+
+There should be one such JSON file for each data file.
+
+The `*_ephys.json` file can be used to store any ephys-specific metadata for the dataset. We recommend storing all setup-related metadata in a dedicated node of the JSON file called `Setup`.
+We recommend using the following keys to describe the setup:
+
+### Data origin metadata
+
+
+{{ MACROS___make_metadata_table(
+   {
+        "InstitutionName":"RECOMMENDED",
+        "InstitutionAddress":"RECOMMENDED",
+        "InstitutionalDepartmentName":"RECOMMENDED",
+   }
+) }}
+
+
+### Setup metadata
+
+
+{{ MACROS___make_metadata_table(
+   {
+        "PowerLineFrequency":"REQUIRED",
+        "Manufacturer":"RECOMMENDED",
+        "ManufacturersModelName":"RECOMMENDED",
+        "ManufacturersModelVersion":"RECOMMENDED",
+        "RecordingSetupName":"RECOMMENDED",
+        "SamplingFrequency":"REQUIRED",
+        "DeviceSerialNumber":"RECOMMENDED",
+        "SoftwareName":"RECOMMENDED",
+        "SoftwareVersions":"RECOMMENDED",
+   }
+) }}
+
+
+### Processing metadata
+
+
+{{ MACROS___make_metadata_table(
+   {
+        "SoftwareFilters":"REQUIRED",
+        "HardwareFilters":"RECOMMENDED",
+   }
+) }}
+
+
+### Additional recording procedure
+
+Furthermore, additional information can be stored about the recording procedure.
+We RECOMMEND to use a dedicated `Procedure` node with the following keys:
+
+<!--Ask reema how she have implemented it-->
+
+<!-- TODO: Yarik replaced Pharmaceuticals with PharmaceuticalName and others for now but we might look to define list of Pharmaceuticals of records with PharmaceuticalDoseAmount and PharmaceuticalDoseUnit -->
+
+{{ MACROS___make_metadata_table(
+   {
+        "PharmaceuticalName":"OPTIONAL",
+        "PharmaceuticalDoseAmount":"OPTIONAL",
+        "PharmaceuticalDoseRegimen":"OPTIONAL",
+        "PharmaceuticalDoseTime":"OPTIONAL",
+        "PharmaceuticalDoseUnit":"OPTIONAL",
+        "SupplementarySignals":"OPTIONAL",
+        "BodyPart":"RECOMMENDED",
+        "BodyPartDetails":"RECOMMENDED",
+        "BodyPartDetailsOntology":"OPTIONAL",
+        "SampleEnvironment":"RECOMMENDED",
+        "SampleEmbedding":"OPTIONAL",
+        "SliceThickness":"OPTIONAL",
+        "SampleExtractionProtocol":"OPTIONAL",
+   }
+) }}
+
+
+### Probes
+
+It is RECOMMENDED to use the following structure to provide more metadata for each probe:
+
+```JSON
+"ProbeContours": {
+   "probe_infoid": {
+      "<my_probe_id>": {
+         "Contour": <list of contour points>,
+         "Unit": "<my spatial unit>"
+      }
+   }
+}
+```
+
+
+Whereas `<my_probe_id>` has to exist also in the `probes.tsv` file and `<list of contour points>` is
+a list of x, y(, z) tuples providing the contour of the probe in the same reference frame as the
+electrode coordinates (see `electrodes.tsv`).
+In case of different units used than in the `electrodes.tsv`, the key `Unit` can be used here to
+provide the spatial unit of the coordinate points.
```

Comment:
This is an accurate depiction of what was discussed in the November 2024, meeting, but in the December 2024 meeting we decided to punt on the countour specification, since it's a bit tricky to include, we might want to refer to ProbeInterface, and it's not critical for spike sorting.